### DISCUSSION TITLE
add support for python 3.10

### DISCUSSION THREAD

**Discussion by rhintze57** [Chia-Network/chia-blockchain#8803] [Ideas]
Seem to have issue running under python 3.10 (so went back to 3.9.7)

tried to modify install.sh (line 85)
-   for V in 39 3.9 38 3.8 37 3.7 3; do
+   for V in 310 3.10 39 3.9 38 3.8 37 3.7 3; do

(some other issues just did not fully test)  could not start farmer.     
Likely this is on your radar already.

**Comment by rhintze57** [comment]
see bug https://github.com/Chia-Network/chia-blockchain/issues/8807

I'm running:
System:    Host: chiaPet  Kernel: 5.14.12 x86_64 bits: 64 Console: tty pts/2 Distro: Slackware 15.0 
Machine:   Type: Desktop Mobo: ASRock model: 990FX Professional serial: <superuser required> UEFI: American Megatrends 
           v: P2.00 date: 10/28/2013

**Comment by BrandtH22** [comment]
@rhintze57 , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
